drivers/adc/ad7616/iio_ad7616.h

```diff
@@ -0,0 +1,37 @@
+/***************************************************************************//**
+* @file   iio_ad7616.h
+* @brief  Header file for AD7616 IIO interface
+********************************************************************************
+* Copyright (c) 2023 Analog Devices, Inc.
+* Copyright (c) 2023 BayLibre, SAS.
+* All rights reserved.
+*
+* This software is proprietary to Analog Devices, Inc. and its licensors.
```

Comment:
Not sure about this copyright and license text, I have zero legal knowledge. I think the ADI SLA should be replicated in full here like in other files.

Reply:
Any updated regarding this?

Reply:
Not on my side, sorry